types/rdf-validate-shacl/src/data-factory.d.ts

```diff
@@ -1,12 +0,0 @@
-import * as RDF from 'rdf-js';
-
-// tslint:disable-next-line no-empty-interface
-interface DataFactory extends RDF.DataFactory, RDF.DatasetCoreFactory {
-}
-
-declare class DataFactory {
-    constructor(impl?: RDF.DataFactory);
-    readonly true: RDF.Literal;
-}
-
-export = DataFactory;
```

Comment:
Doubt that this one is at all important but technically you changed the interface of `SHACLValidator#factory`, effectively removing the `true` property :)

Reply:
Was confused by it: looks to have been removed https://github.com/zazuko/rdf-validate-shacl/pull/62.

Reply:
Sorry, I have not checked the latest 😬